app/javascript/src/styledComponents/Badge.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import classnames from "classnames";
+import PropTypes from "prop-types";
```

Comment:
You don't need a proptype checking if you are using typescript, you can directly use the typescript one.

```suggestion
type Props = {
  color: string;
};
```